packages/application-components/src/components/project-stamp/project-stamp.tsx

```diff
@@ -0,0 +1,63 @@
+import { css } from '@emotion/react';
+import { useIntl } from 'react-intl';
+import { DotIcon } from '@commercetools-uikit/icons';
+import Stamp, { TTone } from '@commercetools-uikit/stamp';
+import messages from './messages';
+
+type TProjectStampProps = {
+  isProductionProject?: boolean | undefined;
+  isSuspended?: boolean | undefined;
+  isExpired?: boolean | undefined;
+  willExpire?: boolean | undefined;
+  daysLeft?: number | undefined;
+};
+
+function ProjectStamp(props: TProjectStampProps) {
```

Comment:
Can we implement some tests for this component?

Reply:
Also some VRTs would be great.

Reply:
Tests - [bb46d1d](https://github.com/commercetools/merchant-center-application-kit/pull/3403/commits/bb46d1dba2c21befe8b0fed16a2ff2af19f90711)
VRT - [44c62cc](https://github.com/commercetools/merchant-center-application-kit/pull/3403/commits/44c62cc00d975e8eb6794f6e70db744e0880e3e6)